src/test/java/seedu/address/model/client/SuggestionTypeTest.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.client;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class SuggestionTypeTest {
+    @Test
```

Comment:
shld add new line before this to standardize

Reply:
Done